dll/win32/browseui/parsecmdline.cpp

```diff
@@ -382,6 +382,9 @@ SHExplorerParseCmdLine(ExplorerCommandLineParseResults * pInfo)
             {
                 // Or just a plain old string.
 
+                if (PathIsDirectoryW(strField))
```

Comment:
This returns true for "C:"? I guess that would be because GetFileAttributes("C:") will actually open the current directory?

Reply:
Yes, it does. A simple test program shows that fact.